styles/designs/carnival/parts/notes/_notes-components.scss

```diff
@@ -140,6 +141,8 @@ $Note--Icon__Container: (
     border-bottom-width: 0.2rem,
     border-bottom-style: solid,
     box-decoration-break: slice,
+    padding-bottom: v-spacing(1),
+    padding-top: v-spacing(1),
```

Comment:
I have added top and bottom paddings to notes containers where they were missing it to prevent lists without margins overlap the note bottom borders. (top padding was added for design consisncy).